libs/entitlement/src/main/java/module-info.java

```diff
@@ -17,6 +17,7 @@
     requires static org.elasticsearch.entitlement.bridge; // At runtime, this will be in java.base
 
     exports org.elasticsearch.entitlement.runtime.api;
+    exports org.elasticsearch.entitlement.runtime.policy;
```

Comment:
What needs this to be exported, isn't all the use within entitlement lib?

Reply:
This was leftover from trying to create a `PolicyManager` in server to pass in if I'm remembering correctly.

Reply:
Removed